packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -204,6 +204,7 @@ export class DumbTriggerManager extends Component {
   async handleSubmit(data = {}) {
     const { client, konnector, saveAccount, vaultClient } = this.props
     const { account } = this.state
+    const cipherId = this.getSelectedCipherId()
```

Comment:
>  Il y a-t-il un risque de faire une sélection à un moment

Je ne comprends pas "faire une sélection" ? S'il n'y a pas de cipher selectionné, `this.getSelectedCipherId()` renverra undefined et c'est supporté dans le KonnectorJob.

Reply:
I think it doesn't change anything. `this.getSelectedCipherId()` will return the same thing, no matter if it's called in its previous place or here. So since the usage is still the same, it's OK.